docs/source/datasets/index.rst

```diff
@@ -3,30 +3,91 @@
 ========
 Datasets
 ========
-Here we cover the simulated datasets, which are analogous to 'real world' administrative records such as tax documents and census surveys, that users can generate using :code:`pseudopeople` to test their PRL methods. 
+Here we cover the simulated datasets, which are analogous to 'real world' administrative records such as tax documents
+and census surveys, that users can generate using :code:`pseudopeople` to test their PRL methods.
+
+The below table offers a list of the datasets that can be generated. Each row of a given dataset represents
```

Comment:
We might consider edits for this part, too, but I don't have the energy to try it right now.

Reply:
Yeah, maybe we can do a second pass later to make everything sound clear and snazzy. But it's probably good to at least get something minimal merged now.